index.js

```diff
@@ -1,18 +1,110 @@
-const readline = require('readline');
+const readline = require("readline");
 const rl = readline.createInterface(process.stdin, process.stdout);
+// Valid string inputs -----------------------------
+const correctAnswer = "yes" || "y" || "ys";
+const wrongAnswer = "no" || "n";
+const highAnswer = "higher" || "h" || "high";
```

Comment:
Another approach to this is to use some Array of values, and check whether the input is included in the Array.
```js
['yes', 'y', 'ys'].includes(correctAnswer)
```